fastify.js

```diff
@@ -121,21 +123,34 @@ function fastify (options) {
   const { logger, hasLogger } = createLogger(options)
 
   // Update the options with the fixed values
-  options.connectionTimeout = options.connectionTimeout || defaultInitOptions.connectionTimeout
-  options.keepAliveTimeout = options.keepAliveTimeout || defaultInitOptions.keepAliveTimeout
+  options.timeouts.connectionTimeout = options.timeouts.connectionTimeout || defaultInitOptions.connectionTimeout
+  options.timeouts.keepAliveTimeout = options.timeouts.keepAliveTimeout || defaultInitOptions.keepAliveTimeout
   options.logger = logger
   options.genReqId = genReqId
   options.requestIdHeader = requestIdHeader
   options.querystringParser = querystringParser
   options.requestIdLogLabel = requestIdLogLabel
   options.disableRequestLogging = disableRequestLogging
   options.ajv = ajvOptions
-  options.clientErrorHandler = options.clientErrorHandler || defaultClientErrorHandler
-  options.exposeHeadRoutes = exposeHeadRoutes
-
-  const initialConfig = getSecuredInitialConfig(options)
+  options.errorManagement.clientErrorHandler = options.errorManagement.clientErrorHandler || defaultClientErrorHandler
+  options.routing.exposeHeadRoutes = exposeHeadRoutes
+
+  const {
+    errorManagement,
+    routing,
+    security,
+    timeouts,
+    ...rest
+  } = options
```

Comment:
I would avoid this desctructure only to use it later

Reply:
Ahh I think the complete logic here would need to be scrapped as per @mcollina 's comment below. I was doing this to kind of use the schema validation in its old format as is